docs/update_and_migration/from_3.3/to_4.0.md

```diff
@@ -168,6 +168,56 @@ php bin/console ibexa:migrations:migrate
 
 ## Update your custom code
 
+### GraphQL
+
+Some GraphQL property names have changed. Adapt your queries according to the table below.
+
+| 3.3 name                                          | 4.0 name                                    |
+|:--------------------------------------------------|:--------------------------------------------|
+| `id` argument                                     | `contentId` argument                        |
+| `_info`                                           | `_contentInfo`                              |
```

Comment:
@mnocon Would it be enough to suggest that _type _info didn't change?
```suggestion
| `_info` content item property                     | `_contentInfo` content item property        |
```

Reply:
I think that's enough, thank you!